src/coreclr/vm/mono/mono_coreclr.cpp

```diff
@@ -1053,27 +1005,6 @@ extern "C" EXPORT_API void EXPORT_CC mono_unity_g_free(void* p)
     free(p);
 }
 
-extern "C" EXPORT_API void EXPORT_CC mono_unity_gc_handles_foreach_get_target (MonoDataFunc callback, void* userData)
-{
-    ASSERT_NOT_IMPLEMENTED;
-}
-
-extern "C" EXPORT_API MonoException* EXPORT_CC mono_unity_loader_get_last_error_and_error_prepare_exception()
-{
-    //ASSERT_NOT_IMPLEMENTED;
-    return NULL;
-}
-
-extern "C" EXPORT_API void EXPORT_CC mono_unity_set_vprintf_func(vprintf_func func)
-{
-    our_vprintf = func;
```

Comment:
I've been leaving this just on the off chance the profiler folks might need a hook into here to get logging done. I guess we can re-add it easily enough if they decide they need something.